tasks/inventory_report/schemas/output.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/inventory_report/schemas/output.json",
+  "title": "inventory_report Lambda Output",
+  "description": "The output for the inventory_report Lambda.",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "required": [
+      "providerId",
+      "collections"
+    ],
+    "properties": {
+      "providerId": {
+        "description": "The unique ID of the provider. Provided by Cumulus.",
+        "type": "string"
+      },
+      "collections": {
+        "description": "Collections within the provider.",
+        "type": "object",
+        "required": [
+          "collectionId",
+          "granules"
+        ],
+        "properties": {
+          "collectionId": {
+            "description": "The unique ID of the collection. Provided by Cumulus.",
+            "type": "string"
+          },
+          "granules": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "required": [
+                "granuleId",
+                "createdAt",
+                "files"

```

Comment:
don't we also need `orcaIngestDate`, `orcaLastUpdate` and `executionId` under granules?

Reply:
Not changed, as per meeting.